Makefile

```diff
@@ -92,14 +92,9 @@ build:
 	#$(_INFO) Building nnictl $(_END)
 	cd tools && python3 setup.py build
 
+	#$(_INFO) Building Training Service tool $(_END)
+	cd src/nni_manager/training_service_tool && python3 setup.py build
 
-# Standard installation target
-# Must be invoked after building
-.PHONY: install
-install: install-python-modules
-install: install-node-modules
-install: install-scripts
-install: install-examples
 install:
```

Comment:
why removing these installs?

Reply:
We shouldn't remove these installs, it may be introduced by merge error and I will recover these lines. Thanks for pointing it out!